internal/agent/install.go

```diff
@@ -249,47 +251,29 @@ func Install(dir ...string) error {
 }
 
 func RunInstall(options map[string]string) error {
+	// TODO: run yip directly
 	utils.SH("elemental run-stage kairos-install.pre")             //nolint:errcheck
```

Comment:
I thought the goal of this PR was to not have "elemental" in the image at all. Are we doing it in steps?

Reply:
we can work on that afterwards in paralell with other work, no need to block this further with that